web-api/openapi/front_office.yaml

```diff
@@ -165,6 +200,40 @@ components:
           type: number
         properties:
           $ref: './schemas/document.yml#/DocumentProperties'
+    StatelessPersonalizedDocumentsRequest:
+      type: object
+      required: [history]
+      properties:
+        history:
+          $ref: './schemas/document.yml#/History'
+        count:
+          $ref: '#/components/schemas/Count'
+        published_after:
+          $ref: './schemas/time.yml#/PublicationDate'
```

Comment:
❓ why do we take a different approach for this request and pass the `count` and `published_after` in the body instead of in the parameter list like we do for other similar endpoints? wouldn't it be better for a user to always have the same way of dealing with a specific parameter?